sparse/core.py

```diff
@@ -553,10 +563,10 @@ def __add__(self, other):
             return self
         if not isinstance(other, COO):
             return self.maybe_densify() + other
-        if self.shape == other.shape:
+        else:  # if self.shape == other.shape:
             return self.elemwise_binary(operator.add, other)
-        else:
-            raise NotImplementedError("Broadcasting not yet supported")
+        # else:
+        #     raise NotImplementedError("Broadcasting not yet supported")
```

Comment:
Code like this should probably be removed.

Reply:
Will do this soon.